test/Ed25519VerificationKey2020.spec.js

```diff
@@ -2,14 +2,18 @@
  * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
  */
 import chai from 'chai';
+import chaiBytes from 'chai-bytes';
```

Comment:
Chai `deep.equal` uses this library under the covers which appears to adequately address the Uint8Array comparison use case.  I'd like to just use `deep.equal` here if possible.

https://github.com/chaijs/deep-eql/blob/v3.0.1/test/new-ecmascript-types.js#L380

Reply:
Sounds good!

Reply:
Switched to deep equal.